### ISSUE TITLE
[Bug]: HC mode > Focus indicator is not visible for pressed toggle button

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#26062]
### Library

React Northstar / v0 (@fluentui/react-northstar)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (12) x64 Intel(R) Xeon(R) E-2176M  CPU @ 2.70GHz
    Memory: 17.39 GB / 31.74 GB
  Browsers:
    Edge: Spartan (44.22621.963.0), Chromium (108.0.1462.54)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://codesandbox.io/s/ovbmp8?file=/example.tsx

### Bug Description

1. Change Windows HC mode into  "aquatic"
2.  Press toggle button(s) into pressed state
3. Navigate to the toggle button which has "pressed" state
 https://react.fluentui.dev/?path=/docs/components-button-togglebutton--default#shape

## Actual Behavior
![image](https://user-images.githubusercontent.com/22620150/208933335-fac2d06f-e8a2-4ec1-b888-dc24107c7928.png)
![image](https://user-images.githubusercontent.com/22620150/208934927-ce3dc7ab-fd2f-473b-8d97-34b938fec9e6.png)

## Expected Behavior
focused button should has visible focus indicator

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
This is the same root cause as the bugs fixed in #26089, but the issue is still present even with that fix, because ToggleButton uses `forced-color-adjust: none` and relies on both `outline-style` and `outline-color` to correctly show the focus outline.
